microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -23,11 +23,11 @@ In the future, the  WebView team plans to change the distribution model for WebV
 
 In preview, the release package contains the following.  
 
-*   [Win32 C/C++ APIs][ReferenceWin3209538]: APIs in the SDK that are expected to remain the same at GA. 
+*   [Win32 C/C++ APIs][ReferenceWin3209538]: APIs in the SDK that are expected to remain the same at GA.  
 
 In preview, the pre-release package contains the following components.  
 
-*   .NET APIs: [WPF][ReferenceWpf09515], [WinForms][ReferenceWinforms09515], and [Core][ReferenceDotnet09538]
+*   .NET APIs: [WPF][ReferenceWpf09515], [WinForms][ReferenceWinforms09515], and [Core][ReferenceDotnet09538]  
 *   Experimental APIs.  For more information, see the [Experimental APIs](#experimental-apis) section.  
 
 ### Experimental APIs  
```

Comment:
Could you make Experimental APIs and Roadmap an ## tag instead of an ### tag.

Reply:
I updated the titles. Can your review please? Thanks.